### PR TITLE
Add bluetooth module

### PR DESCRIPTION
**Pull request by Luka188**
I created this module to connect to my bluetooth headset from polybar and thought it might be useful for others.

### CODE REVIEW

File: polybar-scripts/system-bluetooth/system-bluetooth.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (x70b1) [inline polybar-scripts/system-bluetooth/system-bluetooth.sh]: Please use `#!/bin/sh` and fix all shellcheck errors.

Comment (Luka188) [reply polybar-scripts/system-bluetooth/system-bluetooth.sh]: It now use sh instead of bash and all shellcheck errors are fixed.

**Comment by andrewnicolalde** [comment]
Any updates on this?

**Comment by x70b1** [comment]
Sorry for the late response. Thanks for your work on this topic.

I created [system-bluetooth-bluetoothctl](https://github.com/polybar/polybar-scripts/tree/master/polybar-scripts/system-bluetooth-bluetoothctl) to introduce a bluetooth module.

**Pull request closed without merging**